redirects.csv

```diff
@@ -42,15 +42,15 @@ https://discipleship.crossroads.net/*,https://www.crossroads.net/huddle,301!,mas
 /proxy/content/*,${env:CRDS_CMS_ENDPOINT}/:splat,200!
 /proxy/gateway/*,${env:CRDS_GATEWAY_ENDPOINT}/:splat,200!
 /mycamps*,https://${env:CRDS_ANGULAR_ENDPOINT}/mycamps:splat,200!
-/camps/:campId/*,https://${env:CRDS_ANGULAR_ENDPOINT}/camps/:campId/,200!
+/camps/:campId/*,/camps,301!
 /childcare,https://${env:CRDS_ANGULAR_ENDPOINT}/childcare,200!
 /childcare/*,https://${env:CRDS_ANGULAR_ENDPOINT}/childcare/:splat,200!
 /corkboard*,https://${env:CRDS_ANGULAR_ENDPOINT}/corkboard:splat,200!
 /invoices/*,https://${env:CRDS_INVOICES}/:splat,200!
-/undivided/participant/*,https://${env:CRDS_ANGULAR_ENDPOINT}/undivided/participant/:splat,200!
-/undivided/facilitator/*,https://${env:CRDS_ANGULAR_ENDPOINT}/undivided/facilitator/:splat,200!
+/undivided/participant/*,https://undivided.com,301!
+/undivided/facilitator/*,https://undivided.com,301!
 /mptools/*,https://${env:CRDS_ANGULAR_ENDPOINT}/mptools/:splat,200!
-/profile/personal,profile/personal-info,302!
+/profile/personal,personal-info,301!
```

Comment:
This throws a 404. FWIW, the destination in the spreadsheet specifies a path of `/personalinfo` ... also a 404.